crux-lucene/project.clj

```diff
@@ -14,8 +14,9 @@
   :dependencies [[org.clojure/clojure "1.10.3"]
                  [pro.juxt.crux/crux-core]
 
-                 [org.apache.lucene/lucene-core "8.6.1"]
-                 [org.apache.lucene/lucene-queryparser "8.6.1"]]
+                 [org.apache.lucene/lucene-core "8.9.0"]
+                 [org.apache.lucene/lucene-queryparser "8.9.0"]
+                 [org.apache.lucene/lucene-analyzers-common "8.9.0"]]
```

Comment:
Unless we need this for `crux-lucene` code, could we add this as a test dependency?